src/transformers/models/t5/modeling_t5.py

```diff
@@ -829,7 +829,7 @@ def _init_weights(self, module):
             # See https://github.com/tensorflow/mesh/blob/fa19d69eafc9a482aff0b59ddd96b025c0cb207d/mesh_tensorflow/layers.py#L1624
             module.shared.weight.data.normal_(mean=0.0, std=factor * 1.0)
             if hasattr(module, "lm_head") and not self.config.tie_word_embeddings:
-                module.lm_head.weight.data.normal_(mean=0.0, std=factor * 1.0)
+                module.lm_head.weight.data.normal_(mean=0.0, std=factor * self.config.d_model ** -.5)
```

Comment:
```suggestion
                module.lm_head.weight.data.normal_(mean=0.0, std=factor * 0.05)
```

To align with the origin implementation, let's multiply all lm head inits with 0.05, not sure why this wasn't done back then. 

@Birch-san could you maybe change this here: https://github.com/huggingface/transformers/blob/63864e057fd4ecbf54c77599702873f7be871e65/src/transformers/models/t5/modeling_flax_t5.py#L1512 and here: https://github.com/huggingface/transformers/blob/1b8decb04c246ec8e1c4ba7f2749043d0876d24e/src/transformers/models/t5/modeling_tf_t5.py#L1249 as well?